Fix broken urls and change the url of Coverity Model

matrixise opened the pull request:
Fix #221

Mariatta commented on coverity.rst:
```diff
@@ -150,4 +150,4 @@ Dakshesh Vyas <scan-admin@coverity.com>
 .. _Coverity Connect: http://scan5.coverity.com:8080/
 
-.. _coverity_model.c: https://hg.python.org/cpython/file/tip/Misc/coverity_model.c
+.. _coverity_model.c: https://raw.githubusercontent.com/python/cpython/master/Misc/coverity_model.c
```
I think the url should be https://github.com/python/cpython/blob/master/Misc/coverity_model.c

matrixise replied:
I have updated this url

tiran commented on coverity.rst:
```diff
@@ -150,4 +150,4 @@ Dakshesh Vyas <scan-admin@coverity.com>
 .. _Coverity Connect: http://scan5.coverity.com:8080/
 
-.. _coverity_model.c: https://hg.python.org/cpython/file/tip/Misc/coverity_model.c
+.. _coverity_model.c: https://raw.githubusercontent.com/python/cpython/master/Misc/coverity_model.c
```
+1

ezio-melotti commented:
This PR should be updated.

willingc commented:
Hi @matrixise, Do you want to finish this up or would you prefer closing it? Either one is cool. Thanks.

zware commented on buildslave.rst:
```diff
@@ -196,7 +196,7 @@ Latent slaves
 
 We also support running `latent buildslaves
-<http://docs.buildbot.net/current/manual/cfg-buildslaves.html#latent-buildslaves>`_
+<http://docs.buildbot.net/0.8.14/manual/cfg-buildslaves.html#latent-buildslaves>`_
```
We're now on current buildbot; this page has also been updated in the meanwhile.

matrixise commented:
Hi @Mariatta and @willingc I am really so sorry, I didn't see the notification about this issue/pr. I just updated to the last version of the repo and updated the link for the coverity_model.c file.

When you have time for a review.
